src/common/dict.c

```diff
@@ -388,7 +388,7 @@ KeyHash shd_hash(const void* data, size_t size) {
     unsigned int hash = 0;
     unsigned int i = 0;
 
-    for (i = 0; i < size; data++, i++)
+    for (i = 0; i < size; data_chars++, i++)
```

Comment:
Good catch 😅

Reply:
It was kind of quite lucky. The compilation of the framework failed for Windows because the compiler couldn't estimate the exact size of a single `const void` element in this line. So I was able to find this.